Add docs about default policy for normalized cache (#4512)
apollographql/apollo-kotlin: docs/source/caching/normalized-cache.mdx

## Setting a fetch policy

> After you add a normalized cache to your `ApolloClient` initialization, Apollo Kotlin automatically uses `FetchPolicy.CacheFirst` as default (client-wide) policy for all queries. In order to change it, you need to call **fetch policy** on the client builder:

Reviewer: Slight rephrase suggestion:
```suggestion
After you add a normalized cache to your `ApolloClient` initialization, Apollo Kotlin automatically uses `FetchPolicy.CacheFirst` as the default (client-wide) **fetch policy** for all queries. To change the default, you can call `fetchPolicy` on the client builder:
```

```kotlin
val apolloClient = ApolloClient.Builder()
    .serverUrl("https://...")
    .fetchPolicy(FetchPolicy.NetworkOnly)
    .build()
```

> You can also customize _how_ the cache is used for a particular query by setting a **fetch policy** for that query.

Reviewer: ```suggestion
You can also customize how the cache is used for a particular query by setting a fetch policy for that query.
```
